Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -69,6 +75,28 @@ extension SPAppDelegate {
 
         return true
     }
+
+    func presentSearch() {
+        dismissAllModals(animated: false, completion: nil)
+        sidebarViewController.hideSidebar(withAnimation: false)
+        if navigationController.viewControllers.contains(noteListViewController) {
+            navigationController.popToViewController(noteListViewController, animated: false)
+        }
+
+        DispatchQueue.main.async {
```

Comment:
Do we 100$ need to do this async? (is it for animation purposes?)

Reply:
I did it similar to what we already had in app delegate. I will check if it can be done in a different way.